packages/local-cli/link/__tests__/ios/removeProjectFromProject-test.js

```diff
@@ -35,9 +35,4 @@ describe('ios::addFileToProject', () => {
     const file = removeProjectFromProject(project, filePath);
     expect(project.pbxFileReferenceSection()[file.fileRef]).not.toBeDefined();
   });
-
-  xit('should remove file from PBXContainerProxy', () => {
-    // todo(mike): add in .xcodeproj after Xcode modifications so we can test extra
```

Comment:
Maybe worth marking this test as a todo? In jest 24 with circus runner you can use test.todo for that

Reply:
You are probably right. I didn't know about that feature. 👍